src/fontra/views/fontinfo/fontinfo.js

```diff
@@ -86,6 +93,41 @@ export class FontInfoController extends ViewController {
     const panel = this.panels[this.selectedPanel];
     panel?.handleKeyDown?.(event);
   }
+
+  initActions() {
+    registerActionCallbacks(
+      "action.undo",
+      () => this.doUndoRedo(false),
+      () => this.canUndoRedo(false),
+      () => this.getUndoRedoLabel(false)
+    );
+
+    registerActionCallbacks(
+      "action.redo",
+      () => this.doUndoRedo(true),
+      () => this.canUndoRedo(true),
+      () => this.getUndoRedoLabel(true)
+    );
+  }
+
+  getUndoRedoLabel(isRedo) {
```

Comment:
It would be nicer to delegate getUndoLabel _and_ canUndoRedo to the panel itself. The panel base class can implement them, just like doUndoRedo.

Reply:
Thanks for the feedback. Done.